### ISSUE TITLE
FATAL: Module dnspython missing (python-dnspython)

### ISSUE THREAD

**Issue by 1N3** [1N3/Sn1per#299]
@digitizeddude 

Finally, the last issue that I'm experiencing is that I'm running in to some weird issue with dnscan.py. I'm getting this message when i run a stealth scan using sn1per

BRUTE FORCING DNS SUBDOMAINS VIA DNSCAN (THIS COULD TAKE A WHILE...)
====================================================================================•x2020-11-10x•
FATAL: Module dnspython missing (python-dnspython)

I know that dnspython is installed when i check PIP but it keeps giving me this error.

**Comment by digitizeddude** [comment]
Is the DNSCAN bug a quick fix or is it more involved? How can i help?

**Comment by 1N3** [comment]
Looks like I need to explicitly use python3 for dnscan. To fix, edit your /usr/share/sniper/modes/recon.sh line 43 to the following:

```
python3 $PLUGINS_DIR/dnscan/dnscan.py -d $TARGET -w $DOMAINS_QUICK -o $LOOT_DIR/domains/domains-dnscan-$TARGET.txt -i $LOOT_DIR/domains/domains-ips-$TARGET.txt
```

This will be fixed in v8.9

**Comment by digitizeddude** [comment]
This was driving me bonkers. Spasibo
